src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ErrorAlertUtil.swift

```diff
@@ -0,0 +1,69 @@
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import UIKit
+
+enum ErrorAlertUtil {
+
+	/// This method helps to build a alert for displaying error messages.
+	/// - Parameters:
+	///   - title: The title of the alert. If omitted, it will use the general error title.
+	///   - message: The description of the alert.
+	///   - okTitle: The text of the ok action.
+	///   - secondaryActionTitle: The text of the secondary action, if there is one.
+	///   - hasSecondaryAction: Indicates whether an alert has a secondary action or not.
+	///   - completion: The completion handler for the "ok" action.
+	///   - secondaryActionCompletion: The completion handler for the secondary action.
+	/// - Returns: An alert with either one or two actions, with the specified completion handlers
+	/// and texts.
+	static func setupErrorAlert(
```

Comment:
Do we really need this additional file? Or can we consolidate it and use only extension of UIViewController + Alert.swift and maybe extend this one

Reply:
Good point. I have refactored it into the already existing extension 👍